### DISCUSSION TITLE
飞客茶馆 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#39] [模板讨论]
飞客茶馆 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by xlon72** [comment]
最近都是签到失败
Failed at 18/19 request,
Fail assert: {"re": "您今天还没有签到", "from": "content"} from failed_asserts,
Response Error : OK,
Request URL: https://www.flyert.com/sign.php

安卓app上有10次分享任务可以加吗

**Reply by wjf0214** [reply]
> 最近都是签到失败 Failed at 18/19 request, Fail assert: {"re": "您今天还没有签到", "from": "content"} from failed_asserts, Response Error : OK, Request URL: [flyert.com/sign.php](https://www.flyert.com/sign.php)
> 
> 安卓app上有10次分享任务可以加吗

他网站本身有点问题，就网站登录就经常登录状态没有。
登录那几个请求有时候一次就过，有时会几轮都不行。模板里判断控制比较麻烦。写脚本文件的话可能好点。
签到也是，有时候执行了但是结果又是没有执行。只能失败后多执行几次。

APP上的话我没安装过，有空看看吧。

**Comment by juner922** [comment]
飞客的从3.1号开始就又挂了。。。
<html>
<body>


时间 | 状态 | 日志
-- | -- | --
2023-3-7 15:35:15 | 失败 | Failed at 1/19 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 599: OpenSSL SSL_connect: Connection reset by peer in connection to u.flyert.com:443 ,Request URL: https://u.flyert.com/login
2023-3-4 0:21:09 | 失败 | Failed at 6/19 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 599: OpenSSL SSL_connect: Connection reset by peer in connection to www.flyert.com:443 ,Request URL: {{toUrl}}复制
2023-3-3 13:21:06 | 失败 | Failed at 18/19 request,Fail assert: {"re": "您今天还没有签到", "from": "content"} from failed_asserts,Request URL: https://www.flyert.com/sign.php
2023-3-3 2:20:49 | 失败 | Failed at 2/19 request,Fail assert: {"re": "302", "from": "status"} from success_asserts,Response Error : HTTP 599: OpenSSL SSL_connect: Connection reset by peer in connection to u.flyert.com:443 ,Request URL: https://u.flyert.com/login
2023-3-2 15:20:47 | 失败 | Failed at 6/19 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 599: OpenSSL SSL_connect: Connection reset by peer in connection to www.flyert.com:443 ,Request URL: {{toUrl}}
2023-3-2 4:20:45 | 失败 | Failed at 6/19 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 599: OpenSSL SSL_connect: Connection reset by peer in connection to www.flyert.com:443 ,Request URL: {{toUrl}}
2023-3-1 22:20:42 | 失败 | Failed at 1/19 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 599: OpenSSL SSL_connect: Connection reset by peer in connection to u.flyert.com:443 ,Request URL: https://u.flyert.com/login
2023-3-1 18:20:41 | 失败 | Failed at 1/19 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 599: OpenSSL SSL_connect: Connection reset by peer in connection to u.flyert.com:443 ,Request URL: https://u.flyert.com/login
2023-3-1 16:30:41 | 失败 | Failed at 2/19 request,Fail assert: {"re": "302", "from": "status"} from success_asserts,Response Error : HTTP 599: OpenSSL SSL_connect: Connection reset by peer in connection to u.flyert.com:443 ,Request URL: https://u.flyert.com/login
2023-3-1 16:20:40 | 失败 | Failed at 6/19 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 599: OpenSSL SSL_connect: Connection reset by peer in connection to www.flyert.com:443 ,Request URL: {{toUrl}}


</body>
</html>

**Reply by wjf0214** [reply]
599你网络问题。我这目前正常。看看是不是你网络连接不太好。

**Reply by juner922** [reply]
估计是腾讯云的服务器问题，我现在把镜像换到本地docker了，正常了

**Comment by miaoxiaozhao** [comment]
2024-10-03 00:08:23
日志：Failed at 1/4 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 301: Moved Permanently,
Request URL: https://www.flyert.com/portal.php
